index.js

```diff
@@ -115,13 +115,32 @@ export default function open(params) {
 	createOpenLink(params)();
 }
 
-export function createOpenLink({ provider, ...params }) {
+export async function createOpenLink({ provider, ...params }) {
 	// Returns a delayed async function that opens when executed
+	let mapProvider = provider;
 	if (!provider) {
-		defaultProvider = (Platform.OS === 'ios') ? 'apple' : 'google';
```

Comment:
Is there any reason defaultProvider was used undeclared? I had to declare mapProvider at the top and use variable reassignment. I hope this is clean for you?